### PR TITLE
Feature/particle tuple

### PR DESCRIPTION
**Pull request by whaeck**
Particle pair identifier as a special case for the tuple identifier

### CODE REVIEW

**Comment by whaeck** [comment]
@jlconlin poke ...

**Comment by jlconlin** [comment]
> @jlconlin poke ...

Ouch!

**Comment by whaeck** [comment]
I wasn't aiming for your eye ...

File: src/elementary/ParticlePairID.hpp [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+#include "elementary/Identifier.hpp"
+
+namespace njoy {
+namespace elementary {
+
+  /**
+   *  @class
+   *  @brief The identifier for the special case of a particle pair
+   */
+  class ParticlePairID : public Identifier< ParticlePairID > {
```
Comment (jlconlin) [inline src/elementary/ParticlePairID.hpp:19]: This seems a bit overkill when I think you could just use a c++ alias.

Comment (whaeck) [reply src/elementary/ParticlePairID.hpp:19]: I can't use an alias if I want to restrict the constructor to two particles. I tried reusing the tuple identifier but that didn't really work out as nicely as I had hoped so I went with the Identifier instead.

**Pull request merged**